db/schema.rb

```diff
@@ -142,9 +142,7 @@
     t.index ["user_id"], name: "index_communications_on_user_id"
   end
 
-  create_table "data_migrations", id: false, force: :cascade do |t|
-    t.string "version", null: false
-    t.index ["version"], name: "unique_data_migrations", unique: true
```

Comment:
Why did this disappear?

Reply:
This seems to be something introduced by the rails upgrade in 778fbf6f

Reply:
Weird. So is this column and index in our db or not? Disappearing from the schema without a corresponding migration means our schema and DB are out of sync, right?

Reply:
Just the syntax for this migration seems to have changed. The column and index is still present (line below).